tests/unit_tests/assets/dummy.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
```

Comment:
probably want a better name, like `test_kube_spec.yaml`

Reply:
Agreed with this, maybe even with launch in the name